index.html

```diff
@@ -30,7 +30,9 @@ <h5 class="header col s12 light">{{ site.description }}</h5>
             BOUT++ is primarily designed and tested with reduced plasma fluid
             models in mind, but it can evolve any number of equations, with
             equations appearing in a readable form.
-            <p> Jointly developed by University of York (UK), LLNL, CCFE, DTU, and other international partners
+            <p> Jointly developed by University of York (UK), LLNL, CCFE, DTU, and other international partners.
+	    <p> <strong><font size="7"><center><a href="https://github.com/boutproject/BOUT-dev">Download BOUT++</a></strong>
+	      </center></font>
```

Comment:
Could this maybe be something like:

```
             models in mind, but it can evolve any number of equations, with
             equations appearing in a readable form.
-            <p> Jointly developed by University of York (UK), LLNL, CCFE, DTU, and other international partners.
-	    <p> <strong><font size="7"><center><a href="https://github.com/boutproject/BOUT-dev">Download BOUT++</a></strong>
-	      </center></font>
+            <p> Jointly developed by University of York (UK), LLNL, CCFE, DTU, and other international partners.</p>
         </span></li>
+        <li class="collection-header">
+          <div>
+	        <h3 class="center-align blue-text"><a href="https://github.com/boutproject/BOUT-dev">Download BOUT++</a></h3>
+          </div>
+        </li>
       </ul>
```

This fits in a bit better with the theme